src/middleware.ts

```diff
@@ -21,14 +21,15 @@ export default clerkMiddleware(async (auth, request) => {
 
   try {
     const { userId, sessionClaims } = await auth();
-    const result = ZSessionClaims.safeParse(sessionClaims);
 
-    if (!result.success) {
-      return new Response("Invalid session claims", { status: 400 });
-    }
-
-    const { email } = result.data;
     if (userId) {
+      const result = ZSessionClaims.safeParse(sessionClaims);
```

Comment:
**Question: might as well just use regular parse and then just throw an Exception imho? this should pretty much always be true**